ops/services/app_functions/report_stream_batched_publisher/functions/package.json

```diff
@@ -19,7 +19,7 @@
   },
   "dependencies": {
     "@azure/functions": "^1.2.3",
-    "@azure/identity": "^1.5.2",
+    "@azure/identity": "^2.0.0",
```

Comment:
Also I ran a depcheck against the function app and it showed that azure/identity is unused in the function app. Even going back to the original commit that added it, I don't believe it was actually ever used. I've tested the function app without it and it works!